changes.py

```diff
@@ -0,0 +1,152 @@
+"""
```

Comment:
Let's do `#!/usr/bin/python` here so this script can easily be run directly from a shell.